docs/3.0-Controllers.md

```diff
@@ -240,25 +261,25 @@ status:
     state: Ready
 ```
 
-*state* can be any of the ClaimStatus strings defined in accountclaim_types.go
-*conditions* indicates the last state the account had and supporting details
+* `state` can be any of the ClaimStatus strings defined in accountclaim_types.go
```

Comment:
Link to file `accountclaim_types.go` [https://github.com/openshift/aws-account-operator/blob/master/pkg/apis/aws/v1alpha1/accountclaim_types.go#L84](url)